mmdt/templates/polls/index.html

```diff
@@ -14,26 +15,56 @@ <h3>Latest Poll Questions</h3>
     {% for question in latest_question_list %}
         <div>
             <h4>{{ question.question_text }}</h4>
-            <!-- Display either voting options or results based on user_has_voted -->
             {% if user_has_voted %}
-                <!-- Show results -->
                 {% for choice in question.choice_set.all %}
                     <label>{{ choice.choice_text }}: {{ choice.votes }} vote{{ choice.votes|pluralize }}</label><br>
                 {% endfor %}
             {% else %}
-                <!-- Show voting options -->
                 {% for choice in question.choice_set.all %}
-                    <input type="radio" name="question_{{ question.id }}" id="choice{{ choice.id }}" value="{{ choice.id }}">
-                    <label for="choice{{ choice.id }}">{{ choice.choice_text }}</label><br>
+                    {% if question.is_enabled %}
+                        <input type="radio" name="question_{{ question.id }}" id="choice{{ choice.id }}" value="{{ choice.id }}">
+                        <label for="choice{{ choice.id }}">{{ choice.choice_text }}</label><br>
+                    {% else %}
+                        <input type="radio" name="question_{{ question.id }}" id="choice{{ choice.id }}" value="{{ choice.id }}" disabled>
+                        <label for="choice{{ choice.id }}" style="color: #999;">{{ choice.choice_text }}</label><br>
+                    {% endif %}
                 {% endfor %}
             {% endif %}
+            {% if not question.is_enabled %}
+                <p style="color: rgb(255, 94, 0);">This poll is currently disabled.</p>
```

Comment:
we better not to show the polls, if the poll is disable.

Reply:
- [ ] I was update index.html not to show the polls, if the poll is disable.